project/data_analysis/python/get_window_dr6.py

```diff
@@ -128,9 +130,11 @@ def create_crosslink_mask(xlink_map, cross_link_threshold):
     # Now we create the final window function that will be used in the analysis
     survey_mask.data[dist.data < skip_from_edges_degree] = 0
     survey_mask = so_window.create_apodization(survey_mask, "C1", apod_survey_degree, use_rmax=True)
-    ps_mask = so_map.read_map(d[f"ps_mask_{sv}_{ar}"])
-    ps_mask = so_window.create_apodization(ps_mask, "C1", apod_pts_source_degree, use_rmax=True)
-    survey_mask.data *= ps_mask.data
+
+    if disable_ptsrc_mask == False:
```

Comment:
Default should be `False` then you can write
```python
if d.get("disable_ptsrc_maks", False):
```
You can then remove line 54

Reply:
@xzackli what do you think about the above changes ?

Reply:
Sorry for dropping this PR, but it'll be useful for the upcoming covariance paper as well. I'll make your first suggested change.

Reply:
@thibautlouis are you ok with adding `enable_ptsrc_mask` to every param file?

Reply:
Hi Zack, the only issue I see with this is the fact that we use the point source mask in other part of the analysis, for example in get_alms.py (  ps_map.data *= ps_mask.data) where we add back sources that are below the flux cut threshold. I think this will lead to strange behavior compared to what is expected by the user when he set enable_ptsrc_mask=False. Since this option is only important for a very specific test case, what I would do if I were you would just be to create a unity template (all value set to 1) and pass this as an argument in the dictfile instead of a point source mast. It might look ugly and wasteful, however not using a point source mask in pspipe is such a specific test case that I don't  think it's problematic.